examples/src/main/scala/example/ExampleApi.scala

```diff
@@ -15,39 +16,89 @@ import scala.language.postfixOps
+    )
+
   import auto._
 
   case class Queries(
     @GQLDescription("Return all characters from a given origin")
-    characters: CharactersArgs => URIO[ExampleService, List[Character]],
+    characters: CharactersArgs => URIO[ExampleService, List[CharacterZIO]],
     @GQLDeprecated("Use `characters`")
-    character: CharacterArgs => URIO[ExampleService, Option[Character]]
+    character: CharacterArgs => URIO[ExampleService, Option[CharacterZIO]]
   )
   case class Mutations(deleteCharacter: CharacterArgs => URIO[ExampleService, Boolean])
   case class Subscriptions(characterDeleted: ZStream[ExampleService, Nothing, String])
 
   implicit val originSchema: Schema[Any, Origin]                 = Schema.gen
-  implicit val roleSchema: Schema[Any, Role]                     = Schema.gen
-  implicit val characterSchema: Schema[Any, Character]           = Schema.gen
-  implicit val characterArgsSchema: Schema[Any, CharacterArgs]   = Schema.gen
-  implicit val charactersArgsSchema: Schema[Any, CharactersArgs] = Schema.gen
+  implicit val connectedBySchema: Schema[Any, ConnectedBy]                   = Schema.gen
+  implicit val roleSchema: Schema[Any, Role]                                 = Schema.gen
+  implicit val characterArgsSchema: Schema[Any, CharacterArgs]               = Schema.gen
+  implicit val charactersArgsSchema: Schema[Any, CharactersArgs]             = Schema.gen
+  implicit lazy val characterZIOSchema: Schema[ExampleService, CharacterZIO] = gen
+  implicit val queriesSchema: Schema[ExampleService, Queries]                = gen
 
   val api: GraphQL[ExampleService] =
     graphQL(
       RootResolver(
         Queries(
-          args => ExampleService.getCharacters(args.origin),
-          args => ExampleService.findCharacter(args.name)
+          args => ExampleService.getCharacters(args.origin).map(_.map(character2CharacterZIO)),
+          args => ExampleService.findCharacter(args.name).map(_.map(character2CharacterZIO))
         ),
         Mutations(args => ExampleService.deleteCharacter(args.name)),
         Subscriptions(ExampleService.deletedEvents)
       )
     ) @@
-      maxFields(200) @@               // query analyzer that limit query fields
+      maxFields(300) @@               // query analyzer that limit query fields
       maxDepth(30) @@                 // query analyzer that limit query depth
       timeout(3 seconds) @@           // wrapper that fails slow queries
       printSlowQueries(500 millis) @@ // wrapper that logs slow queries
       printErrors @@                  // wrapper that logs errors
-      apolloTracing                   // wrapper for https://github.com/apollographql/apollo-tracing
-
+      apolloTracing @@                // wrapper for https://github.com/apollographql/apollo-tracing
+      DeferSupport.defer @@    // wrapper that enables @defer directive support
+      DeferSupport.stream      // wrapper that enables @stream directive support
```

Comment:
If streaming is not supported, how about deleting all references to it? I think it's quite confusing to see it here and there.